mantidproject/mantid
# Make VSI popups modal

**To test:**

Reproduce instructions in the ticket. Check that all but edit button next to the "Axis Aligned" checkbox are now modal. That specific one cannot be fixed quickly before 3.9 release.

Fixes #18733.



---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
